torch/_refs/__init__.py

```diff
@@ -5208,13 +5234,13 @@ def exponential(self, rate=1, generator=None):
 bitwise_or_ = _make_inplace(bitwise_or)
 bitwise_right_shift_ = _make_inplace(bitwise_right_shift)
 bitwise_xor_ = _make_inplace(bitwise_xor)
-ceil_ = _make_inplace(ceil)
-clamp_ = _make_inplace(clamp)
+ceil_ = _make_inplace(ceil, shape_preserving=shape_preserving_default)
+clamp_ = _make_inplace(clamp, shape_preserving=shape_preserving_default)
 clamp_min_ = _make_inplace(clamp_min)
 clamp_max_ = _make_inplace(clamp_max)
 conj_physical_ = _make_inplace(conj_physical)
 copysign_ = _make_inplace(copysign)
-cos_ = _make_inplace(cos)
+cos_ = _make_inplace(cos, shape_preserving=shape_preserving_default)
```

Comment:
no unfortunately, because if input dtype is integral, output dtype will be float